src/spatial/detail/ArborX_ExpandHalfToFull.hpp

```diff
@@ -34,12 +34,14 @@ void expandHalfToFull(ExecutionSpace const &space, Offsets &offsets,
   Kokkos::parallel_for(
       "ArborX::Experimental::HalfToFull::count",
       Kokkos::RangePolicy(space, 0, n), KOKKOS_LAMBDA(int i) {
+        int local_update = 0;
         for (int j = offsets_orig(i); j < offsets_orig(i + 1); ++j)
         {
-          int const k = indices_orig(j);
-          Kokkos::atomic_inc(&offsets(i));
+          ++local_update;
+          auto const k = indices_orig(j);
```

Comment:
Why auto for k ( we are not doing that for j)

Reply:
Not any particular reason. I don't think it matters.